src/main/resources/scripts/anomalyDetectionTesting.txt

```diff
@@ -0,0 +1,79 @@
+##===========SETUP===============##
```

Comment:
Why do we have scripts in `txt` file? We should have it in `.sh`

Reply:
My 2c here: unless we want this to be executable (which we don’t and probably can’t via git) it doesn’t matter.   I’d probably prefer no extension to txt.  I’m ok with it being sh.

Reply:
In new changes I had change the file extension to .sh. I did that because in future I will simple pull this via git command line and in aws ec2 run instance command, there is a parameter name --userdata in which we pass the file and it will do all this installation stuff while creating the instance itself. Right now, I dropped that plan because of OSB support not there for extensions. If that support comes then I will turn this all infra into one click feature, where you will just click a button to run pipeline in jenkins and the end result would be on opensearch dashboard reflecting the whole summary of performance.